### ISSUE TITLE
Double-click event of electron is too short in notebook, making it difficult to edit a markdown cell

### ISSUE THREAD

**Issue by rodrigosf672** [posit-dev/positron#10255]
```
Positron Version: 2025.12.0 build 9
Code - OSS Version: 1.105.0
Commit: e5c70af403b8af0ea2167ede8ae47d45bea793c3
Date: 2025-10-31T07:19:30.891Z
Electron: 37.6.0
Chromium: 138.0.7204.251
Node.js: 22.19.0
V8: 13.8.258.32-electron.0
OS: Darwin arm64 25.0.0
```

### Expected behavior
User should be able to easily edit a markdown cell, easily and accessibly. In VSC/JupyterLab/Colab notes, user can do so by double-clicking OR by hitting 'Enter'.

### Current behavior
In new notebooks, 'Enter' does NOT work (as designed?). As a result, user has to double-click in order to edit a markdown cell. However, the timing of a double-click event in electron is too short. Hence, user has to double click unreasonably quickly in order to be able to edit a cell. 

<img width="948" height="107" alt="Image" src="https://github.com/user-attachments/assets/c4140a18-ca13-4891-b7d5-5b51dd0749b0" />

**Comment by rodrigosf672** [comment]
In addition, new markdown cell does NOT go into edit mode by default.

**Comment by cindyytong** [comment]
We could not reproduce, this might have been addressed in a recent improvement. make followup ticket to update message for editing with enter

**Comment by rodrigosf672** [comment]
> We could not reproduce, this might have been addressed in a recent improvement. make followup ticket to update message for editing with enter

Details:
`2025.12.0 build 35`
1. Double-clicking timing is reasonable now!
2. Enter does work! 

@cindyytong For reference, message from old notebooks was the following. 

<img width="486" height="40" alt="Image" src="https://github.com/user-attachments/assets/6b2f60ed-1e0f-4e61-b73d-4bf5c1681d2d" />

For the new message, we could have something like:

`Empty markdown cell. Double-click or press Enter to edit.`
